code/hw_5_pandas.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+"""
+This is just a copy from gcollab, a pale decoy, boring fake
+"""
+""" 1 """
+import pandas as pd
+
+
+def read_gff(annotation_file: str):
+    col_names = ['chromosome', 'source', 'type', 'start', 'end', 'score', 'strand', 'phase', 'attributes']
+    annotation_df = pd.read_csv(annotation_file, sep='\t', comment='#', names=col_names)
+
+    return annotation_df
+
+
+def read_bed6(alignment_file: str):
+    col_names = ['chromosome', 'start', 'end', 'name', 'score', 'strand']
+    alignment_df = pd.read_csv(alignment_file, sep='\t', comment='#', names=col_names)
+
+    return alignment_df
+
+
+def check_intersection(start_x, end_x, start_y, end_y): # checks if a contig completely intersects with some rRNA gene
+    if start_y <= start_x and end_y >= end_x:
+        return 'YES'
+    else:
+        return 'NOPE'
+
+
+if __name__ == '__main__':
+    rrna_df = read_gff('rrna_annotation.gff')

```

Comment:
Очень круто, что ты загрузила данные в репозиторий и положила их в отдельную папку, это очень хорошая практика. Но раз в репозитории есть данные, то код должен уметь с ними работать "из коробки", а-ля клонировали, запустили ноутбук ничего не изменяя и всё.
```suggestion
    rrna_df = read_gff('../data/rrna_annotation.gff')

```